src/cmd-buildupload

```diff
@@ -54,6 +54,11 @@ def cmd_upload_s3(args):
             for arch in builds.get_build_arches(args.build):
                 s3_upload_build(args, builds.get_build_dir(args.build, arch),
                                 f'{args.build}/{arch}')
+            # upload the release.json directly if running a multi-arch build. Legacy
+            # builds inherently upload this file by working on the entire build dir
+            s3_cp(args, os.path.join(builds.get_build_dir(args.build), 'release.json'),
+                  os.path.join(args.build, 'release.json'),
```

Comment:
The `release.json` is an FCOS pipeline thing though, not really cosa specific. E.g. a local `buildupload` would fail on this. (If we start creating it as part of `cosa build`, that's a different story). How about something a bit more agnostic, e.g.:

```python
# if there's anything else in the build dir, just upload it too, e.g. pipelines
# might inject additional metadata
for f in os.listdir(f'builds/{args.build}'):
    # arches already uploaded higher up
    if f in builds.get_build_arches(args.build):
        continue
    s3_cp(args, f'builds/{args.build}/{f}', f'{args.build}/{f}')
```

?